test/Sentry.Tunnel.Tests/Sentry.Tunnel.Tests.csproj

```
@@ -0,0 +1,47 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net5.0;netcoreapp3.1;netcoreapp2.1</TargetFrameworks>
```

In case you take the change for the library:

```suggestion
    <TargetFrameworks>netcoreapp3.1;netcoreapp2.1</TargetFrameworks>
```